### DISCUSSION TITLE
How to stop the automatic scaling if adding a point on click

### DISCUSSION THREAD

**Discussion by pluto-dev** [Live-Charts/LiveCharts2#808] [Q&A]
I'm using the [example](https://lvcharts.com/docs/winui/2.0.0-beta.330/samples.events.addPointOnClick) "Add Point On Click". Is there a way to add a point without scaling down or up?

**Comment by Kevin-Seiden** [comment]
When creating the Y axis, you have to define ICartesianAxis.MinLimit and ICartesianAxis.MaxLimit (technically IPlane but ICartesianAxis derives from IPlane).  If you leave them null (which is the default) it will scale to the highest/lowest point.
